website/blog/2022/03/02/apisix-integration-graphql-plugin.md

```diff
@@ -0,0 +1,347 @@
+---
+title: "How to Use GraphQL with API Gateway Apache APISIX"
+authors:
+  - name: "JohnChever"
+    title: "Author"
+    url: "https://github.com/Chever-John"
+    image_url: "https://avatars.githubusercontent.com/u/43690894?v=4"
+  - name: "Fei Han"
+    title: "Technical Writer"
+    url: "https://github.com/hf400159"
+    image_url: "https://avatars.githubusercontent.com/u/97138894?v=4"
+keywords: 
+- Apache APISIX
+- API Gateway
+- GraphQL
+- Ecosystem
+description: This article introduces the features of Apache APISIX and GraphQL, and how to use the API gateway Apache APISIX to proxy GraphQL requests, and proposes solutions to solve the pain points of practical scenarios.
+tags: [Technology,Ecosystem,Security]
```

Comment:
The Security tag is meant for CVE news. Please delete this tag to avoid confusion.
```suggestion
tags: [Technology,Ecosystem]
```